trackpy/feature.py

```diff
@@ -274,24 +281,29 @@ def _refine(raw_image, image, radius, coords, max_iterations,
         mass[feat] = neighborhood.sum()
         if not characterize:
             continue  # short-circuit loop
-        Rg[feat] = np.sqrt(np.sum(r_squared_mask(radius, ndim) *
-                                  neighborhood) / mass[feat])
+        if radius[1:] == radius[:-1]:
```

Comment:
Probably the line numbering changed, I can't find the line you are referring to. It should be a tuple, except for the use inside `_numba_refine`, where everything is an ndarray (but only in this function).